src/utils/signaling.js

```diff
@@ -849,12 +846,37 @@ Signaling.Standalone.prototype.helloResponseReceived = function(data) {
 			return
 		}
 
+		this.helloResponseErrorCount++
+
+		if (this.signalingConnectionError === null && this.helloResponseErrorCount < 5) {
+			this.signalingConnectionError = showError(t('spreed', 'Failed to establish signaling connection. Retrying …'), {
```

Comment:
Is this part missing the hideToast call before?

Reply:
No, because it will be shown only if `this.signalingConnectionError === null`, and if it is null you can not call `hideToast()` on it :-)